_literate/EX-breastcancer/tutorial.jl

```diff
@@ -200,11 +199,11 @@ for m in models(matching(X, y))
 end
 
 #Adding labels and legend to the ROC-AUC curve
-xlabel("False Positive Rate")
-ylabel("True Positive Rate")
-legend(loc="best", fontsize="xx-small")
-title("ROC curve")
-plt.savefig(joinpath(@OUTPUT, "breastcancer_auc_curve.svg")) # hide
+xlabel!("False Positive Rate")
+ylabel!("True Positive Rate")
+#legend(loc="best", fontsize="xx-small")
+#title("ROC curve")
```

Comment:
Code commented out.

Reply:
Sorry should indeed be removed and I should add the title. Excuse me for this as I should have given myself more time especially while working on the end-to-end examples.

Reply:
No big deal. I always miss things. That's why we have a review process :wink:

Reply:
Thank you for your kindness 🙏.